src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs

```diff
@@ -17,6 +17,10 @@ public class HtmlCodeSnippetRenderer : HtmlObjectRenderer<CodeSnippet>
     {
         private readonly MarkdownContext _context;
         private const string tagPrefix = "snippet";
+        private const string warningMessageId = "codeIncludeNotFound";
+        private const string defaultWarningMessage = "It looks like the sample you are looking for has moved!  Rest assured we are working on resolving this.";
```

Comment:
The wording is good for docs user, but not well applies to community user. What about "It looks like the sample you are looking for does not exist." as default message?

Reply:
Oh, so for en-us build, template will also pass the token to markdown engine? If that, I think it's make sense to use the message you mentioned as the default string.

Reply:
Yes, it exists: https://github.com/Microsoft/templates.docs.msft/blob/master/ContentTemplate/token.json#L75